tests/integration_tests/functional/test_mmds.py

```diff
@@ -396,7 +416,11 @@ def test_mmds_dummy(test_microvm_with_ssh):
 
 
 def test_guest_mmds_hang(test_microvm_with_ssh, network_config):
-    """Test the MMDS json response."""
+    """
+    Test the MMDS json response.
```

Comment:
Test the MMDS json response when the declared content length is larger than the actual length of the request.